packages/client/webapp/src/components/sidebar-panel/documents-sidebar-panel.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<main>
+		<header>Documents space</header>
+		<section>
+			Project Documents:
+			<div class="document-list-container">
+				<div
+					v-for="doc in documents"
+					:key="doc._gddid"
+					class="doc-link"
+					:class="{ active: doc._gddid === docID }"
+					@click="openDocumentPage(doc)"
+				>
+					<span>{{ formatTitle(doc) }}</span>
+					<span class="doc-delete-btn" @click.stop="removeDoc(doc)">
+						<i class="fa fa-fw fa-close" />
+					</span>
```

Comment:
We are using IBM Carbon Icons, not font-awesome.

Reply:
done